pkg/cmd/server/server.go

```diff
@@ -0,0 +1,172 @@
+package server
```

Comment:
I am not sure I quite follow why the files are in such a varied locations but hopefully there is an easy to understand rationale for it which will be more apparent as @pmorie gives us overview of the k8s world.